logzio/datasource_metrics_account_test.go

```
@@ -0,0 +1,128 @@
+package logzio
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/logzio/logzio_terraform_provider/logzio/utils"
+	"os"
+	"regexp"
+	"strconv"
+	"testing"
+)
+
+func TestAccDataSourceMetricsAccount(t *testing.T) {
+	resourceName := "logzio_metrics_account.metrics_account_datasource"
+	dataSourceName := "data.logzio_metrics_account.metrics_account_datasource_by_id"
+	accountId, _ := strconv.ParseInt(os.Getenv(envLogzioAccountId), utils.BASE_10, utils.BITSIZE_64)
```

why this error is ignored?